helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/values.yaml

```diff
@@ -11,9 +11,11 @@ tolerations: []
 affinity: {}
 resources:
   requests:
-    memory: 256Mi
+    cpu: "0.5"
+    memory: "256Mi"
   limits:
-    memory: 2048Mi
+    cpu: "4"
```

Comment:
4 is a rather high value. I understand indexing process may need that but does the mediation pod also needs that?

Reply:
yes for now let's move on as is
OPSEXP-1976